### PR TITLE
fixing MC-119 : clarifying approximations about max. heap size

### PR DESCRIPTION
**Pull request by erosb**

### CODE REVIEW

File: src/docs/asciidoc/cache_size.adoc [hunk 1]
```diff
@@ -51,5 +51,9 @@ count, map count in each member, and `hazelcast.mc.cache.max.size` setting.
 |
 |===
 
+NOTE: in case the cluster has a lot of connected clients (especially with near-caches), or the number of other data structures is significant, then these will consumer additional heap space, and `hazelcast.mc.cache.max.size` will have to be adjusted accordingly. The above reference numbers were taken by benchmarking a cluster with no clients and no distributed objects other than maps.
```
Comment (emre-aydin) [inline src/docs/asciidoc/cache_size.adoc:54]:
```suggestion
NOTE: When the cluster has a lot of connected clients (especially with near-caches) or the number of other data structures is significant, these will consume additional heap space. In this case, `hazelcast.mc.cache.max.size` need to be adjusted accordingly. The above reference numbers were taken by benchmarking a cluster with no clients and no distributed objects other than maps.
```

**Review by emre-aydin — APPROVED** [review]

File: src/docs/asciidoc/cache_size.adoc [hunk 2]
```diff
@@ -51,5 +51,9 @@ count, map count in each member, and `hazelcast.mc.cache.max.size` setting.
 |
 |===
 
+NOTE: When the cluster has a lot of connected clients (especially with near-caches) or the number of other data structures is significant, these will consume additional heap space. In this case, `hazelcast.mc.cache.max.size` need to be adjusted accordingly. The above reference numbers were taken by benchmarking a cluster with no clients and no distributed objects other than maps.
```
Comment (Serdaro) [inline src/docs/asciidoc/cache_size.adoc:54]: ... clients (especially with Near Caches) or the ...

.. heap space. In this case, you need to adjust `hazelcast.mc.cache.max.size` accordingly. The above ...

File: src/docs/asciidoc/cache_size.adoc [hunk 3]
```diff
@@ -51,5 +51,9 @@ count, map count in each member, and `hazelcast.mc.cache.max.size` setting.
...
+
+NOTE: In cases when adjusting `hazelcast.mc.cache.max.size` is not sufficient, you may consider turning off statistics collection for some of your distributed objects with their respective `"statistics-enabled"` configuration settings, so that Management Center will have to store less data in-memory.
+
```
Comment (Serdaro) [inline src/docs/asciidoc/cache_size.adoc:57]: ... configuration settings, so that Management Center stores less in-memory data.

**Comment by erosb** [comment]
Thanks for the review @Serdaro , I addressed your comments.

**Pull request merged**